gap_nrf51.go

```
@@ -72,6 +72,12 @@ func (a *Advertisement) Stop() error {
 	return makeError(errCode)
 }
 
+// SetServiceData sets the service data for the advertisement.
+func (a *Advertisement) SetServiceData(sd []ServiceDataElement) error {
+	// TODO: implement
+	return nil
```

As long as these are unimplemented, I'd say they need to return an error.

---

Good point. Changed everywhere that it is not yet implemented.

---

Oops I missed that one. Done.